examples/embeddable_examples/public/react_embeddables/saved_book/saved_book_editor.tsx

```diff
@@ -0,0 +1,208 @@
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.authorLabel', {
+              defaultMessage: 'Author',
+            })}
+          >
+            <EuiFieldText
+              value={authorName}
+              onChange={(e) => attributesManager.authorName.next(e.target.value)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.titleLabel', {
+              defaultMessage: 'Title',
+            })}
+          >
+            <EuiFieldText
+              value={bookTitle}
+              onChange={(e) => attributesManager.bookTitle.next(e.target.value)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.pagesLabel', {
+              defaultMessage: 'Number of pages',
+            })}
+          >
+            <EuiFieldNumber
+              value={numberOfPages}
+              onChange={(e) => attributesManager.numberOfPages.next(+e.target.value)}
+            />
+          </EuiFormRow>
+          <EuiFormRow
+            label={i18n.translate('embeddableExamples.savedBook.editor.synopsisLabel', {
+              defaultMessage: 'Synopsis',
+            })}
+          >
+            <EuiTextArea
+              value={synopsis}
+              onChange={(e) => attributesManager.bookSynopsis.next(e.target.value)}
+            />
+          </EuiFormRow>
+        </EuiFormControlLayout>
+      </EuiFlyoutBody>
+      <EuiFlyoutFooter>
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty iconType="cross" onClick={onCancel} flush="left">
+              {i18n.translate('embeddableExamples.savedBook.editor.cancel', {
+                defaultMessage: 'Cancel',
+              })}
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="m" alignItems="center" responsive={false}>
+              {isCreate && (
+                <EuiFlexItem grow={false}>
+                  <EuiSwitch
+                    label={i18n.translate('embeddableExamples.savedBook.editor.addToLibrary', {
+                      defaultMessage: 'Save to library',
+                    })}
+                    checked={addToLibrary}
+                    onChange={() => setAddToLibrary(!addToLibrary)}
+                  />
+                </EuiFlexItem>
+              )}
+              <EuiFlexItem grow={false}>
+                <EuiButton onClick={() => onSubmit(addToLibrary)} fill>
+                  {isCreate
+                    ? i18n.translate('embeddableExamples.savedBook.editor.create', {
+                        defaultMessage: 'Create book',
+                      })
+                    : i18n.translate('embeddableExamples.savedBook.editor.save', {
+                        defaultMessage: 'Apply changes',
```

Comment:
How about changing to "Keep changes" since changes are all ready applied

Reply:
Good idea. Resolved in [541b84b](https://github.com/elastic/kibana/pull/182523/commits/541b84bdee44c2d4018f7c12720770bba34681b4)